src/stories/theme/_dropdown.css

```diff
@@ -0,0 +1,11 @@
+.ck.ck-dropdown .ck-dropdown__button {
```

Comment:
It looks like `.ck-dropdown__panel` can receive a `transform: translateY(...)` to position the menu as desired. There's an additional class (i.e. `.ck-dropdown__panel_se`) to help inform the direction for the transform. In addition, Garden's show/hide menu animations should be getting applied.

Also, border radius should be set to `!important` to prevent odd squaring-off when menu is shown.

Reply:
I've updated the relative positioning, styling, and animation for the dropdown. There only seems to be an "up/down" directionality so I've only included those animations for now.

There will need to be some additional style tweaks as I fix the toolbar spacing in following PRs.